### ISSUE TITLE
Need to pin some of our dependencies

### ISSUE THREAD

**Issue by jonathandturner** [nushell/nushell#430]
Trying to build a recent checkout, some folks (myself included) are seeing this:

```
error[E0596]: cannot borrow `self` as mutable, as it is not declared as mutable
  --> C:\Users\joturner\.cargo\registry\src\github.com-1ecc6299db9ec823\futures-async-stream-0.1.0-alpha.3\src\stream.rs:43:20
   |
42 |     fn poll_next(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
   |                  ---- help: consider changing this to be mutable: `mut self`
43 |         let this = self.project();
   |                    ^^^^ cannot borrow as mutable

error: aborting due to previous error
```

May need to go and pin the version for `futures-async-stream`

**Comment by androbtech** [comment]
Can confirm.

**Comment by jonathandturner** [comment]
Fixed by #432
